TARGETS

```diff
@@ -8,10 +8,13 @@ TEST_RUNNER = REPO_PATH + "buckifier/rocks_test_runner.sh"
 
 rocksdb_compiler_flags = [
     "-fno-builtin-memcmp",
-    "-DROCKSDB_PLATFORM_POSIX",
-    "-DROCKSDB_LIB_IO_POSIX",
+    "-DROCKSDB_BACKTRACE",
     "-DROCKSDB_FALLOCATE_PRESENT",
+    "-DROCKSDB_JEMALLOC",
```

Comment:
In the past I tried it but it failed with internal ASAN build and I didn't find a way to fix it. Make sure it is not a problem this time.

Reply:
Sandcastle do fail for missing jemalloc.h. Maybe we need to way to tell if jemalloc presents in the buck script.

Reply:
Sent #4489 separately for the TARGETS file change.